agences-regionales-sante/centre-val-de-loire/2020-03-13.yaml

```diff
@@ -0,0 +1,35 @@
+date: 2020-03-12
+source:
+  nom: ARS Centre - Val de Loire
+  url: https://www.centre-val-de-loire.ars.sante.fr/system/files/2020-03/2020-03-13-bulletin%20information%20COVID-19.pdf
```

Comment:
Hi @CAEck, this url does not work. Do you have any link to a valid source for 2020-03-12? I don't find any.

FR: Bonjour @CAEck cette url ne fonctionne pas. Disposes-tu d'une source d'info valide pour cette date ? J'ai cherché je ne trouve rien.

Reply:
Un nouveau bulletin a été ajouté au 2020-03-14 et le PDF du 2020-03-13 a été effacé (!) Il faut les télécharger quotidiennement...

Reply:
Des fois, j’arrive à trouver les bulletins dans les sites des départements:

- 08/03: http://www.indre.gouv.fr/content/download/23261/163524/file/Bulletin%20d'information%20Covid-19%20-%20n%C2%B01%20-%2008-03-20%20_1.pdf
- 10/03: http://www.indre.gouv.fr/content/download/23274/163576/file/Bulletin%20d'information%20Covid-19%20-%20n%C2%B02%20-%2010-03-20%20.pdf
- 11/03: http://www.indre.gouv.fr/content/download/23298/163705/file/Bulletin_Covid19_11mars_2.pdf
-12/03: http://www.indre.gouv.fr/content/download/23306/163759/file/2020-03-12-bulletin%20information%20COVID-19.pdf
- 13

Reply:
Hello @jdesboeufs : est-il possible d'héberger les bulletins PDF dans le repo ou ailleurs ? Vu que certains sont bulletins sont supprimés, on perd le l'info.

Reply:
Il faut aller les chercher sur archive.org, dans la mesure du possible.
Tapes l'URL dans la barre Wayback Machine, et choisis la date de snapshot qui convient.
Il est aussi possible de déclencher un snapshot via cette page : https://web.archive.org/save/

Reply:
Les derniers YAML de Santé public France sont comme ça.

Reply:
Oui, mais là le problème se pose pour les fichiers PDF des ARS. Or wayback machine ne stocke pas (à ma connaissance) les fichiers PDF. D'où ma question, peut-on stocker les PDF qqpart ? :wink:

Reply:
Si si !
Exemple :
https://web.archive.org/web/20200315215824/http://www.indre.gouv.fr/content/download/23306/163759/file/2020-03-12-bulletin%20information%20COVID-19.pdf

Faut utiliser la page save, c'est un peu lent cela dit.

Reply:
Hello
Du coup toujours dans l'attente d'une action de ma part ou pas ?
Bon confinement à tous ^^